packages/next/build/webpack/loaders/next-serverless-loader/utils.ts

```diff
@@ -85,7 +89,10 @@ export function getUtils({
     defaultRouteMatches = dynamicRouteMatcher(page) as ParsedUrlQuery
   }
 
-  function handleRewrites(req: IncomingMessage, parsedUrl: UrlWithParsedQuery) {
+  function handleRewrites(
+    req: IncomingMessage | WebIncomingMessage,
```

Comment:
not blocking: we can have a type for this `req`, `res` type later and share them across modules?

Reply:
> we can have a type for this req, res type later

Could you explain this? You mean get them into one type? That sounds good to me.